api/external/nodes/manager/manager.proto

```diff
@@ -42,6 +56,13 @@ service NodeManagerService {
 			action: "infra:nodeManagers:get"
 		};
 	};
+
+	/*
+	Update a Node Manager
+
+	Update a node manager's metadata, such as its name, associated credential id, or data.
+	This is a PUT operation and it overwrites ALL of the existing node manager metadata. Include all fields, because a PUT operation overwrites any missing fields to empty ("").
+	*/	
 	rpc Update(NodeManager) returns (google.protobuf.Empty) {
```

Comment:
![Screen Shot 2020-02-14 at 11 02 04 AM](https://user-images.githubusercontent.com/22962/74559525-7e4d6700-4f19-11ea-9ca0-b68d78b8606a.png)

Not really related to your PR, but the output in the UI when an endpoint returns `google.protobuf.Empty` looks strange.

Reply:
It really does!